crate/operator/utils/kubeapi.py

```diff
@@ -162,7 +162,7 @@ async def get_public_host(core: CoreV1Api, namespace: str, name: str) -> str:
         except ApiException:
             pass
 
-        await asyncio.sleep(BACKOFF_TIME / 2)
+        await asyncio.sleep(10)
```

Comment:
Maybe we can set this to something even lower? Doesn't cost us much to make this API call to kubernetes when retrying...

Reply:
Indeed. What about 5s?